conf/services/register-charity-hmrc.yml

```diff
@@ -3,24 +3,10 @@ serviceHeaderName: Get recognition from HMRC for your charity
 serviceDescription: |
   Charities use this service to register their details with HMRC. Only once they are successfully registered as a charity are they able to get tax back on things like Gift Aid donations.
 serviceDomain: www.tax.service.gov.uk
-serviceUrl: /register-charity-hmrc.yml
+serviceUrl: /register-charity-hmrc
```

Comment:
service url  **https://www.tax.service.gov.uk/register-charity-hmrc** goes to page not found page... need to fix this?